.github/workflows/labels.yml

```diff
@@ -0,0 +1,14 @@
+name: Required Labels
+
+on:
+  pull_request:
+    types: opened
```

Comment:
`opened` only is probably not good, because it will fail if a label isn't applied, and then never succeed even if labels are added later. Should make it run when opened and when labels are modified maybe?

Reply:
Extend types. Now, it will re-run job on adding/removing labels